Docs/Configuration.tex

```diff
@@ -3186,98 +3208,101 @@ \subsection{Boot Properties}\label{miscbootprops}
 
   \begin{itemize}
   \tightlist
-  \item \texttt{Default} --- this is the default option, and it lets OpenCore built-in
-  boot picker to loads the default boot option as specified in
+  \item \texttt{Default} --- this is the default option, and it lets the built-in OpenCore
+  boot picker to load the default boot option as specified in the
   \href{https://support.apple.com/HT202796}{Startup Disk} preference pane.
-  \item \texttt{ShowPicker} --- this option forces picker to show. Normally it can be
-  achieved by holding \texttt{OPT} key during boot. Setting \texttt{ShowPicker} to
+  \item \texttt{ShowPicker} --- this option forces the boot picker to be displayed. This can
+  typically be achieved by holding the \texttt{OPT} key during boot. Setting \texttt{ShowPicker} to
   \texttt{true} will make \texttt{ShowPicker} the default option.
   \item \texttt{ResetNvram} --- this option erases certain UEFI variables and is
-  normally executed by holding the \texttt{CMD+OPT+P+R} key combination during boot.
-  Another way to erase UEFI variables is to choose \texttt{Reset NVRAM} in the picker.
+  normally executed by holding down the \texttt{CMD+OPT+P+R} key combination during boot.
+  Another way to erase UEFI variables is to choose \texttt{Reset NVRAM} in the boot picker.
   This option requires \texttt{AllowNvramReset} to be set to \texttt{true}.
-  \item \texttt{BootApple} --- this options performs booting to the first found Apple
-  operating system unless the default chosen operating system is already made by Apple.
-  Hold \texttt{X} key to choose this option.
-  \item \texttt{BootAppleRecovery} --- this option performs booting to Apple operating
-  system recovery. Either the one related to the default chosen operating system,
-  or first found in case default chosen operating system is not made by Apple or has no
-  recovery. Hold \texttt{CMD+R} key combination to choose this option.
+  \item \texttt{BootApple} --- this options performs booting to the first Apple
+  operating system found unless the chosen default operating system is one from Apple.
+  Hold the \texttt{X} key down to choose this option.
+  \item \texttt{BootAppleRecovery} --- this option performs booting into the Apple operating
+  system recovery partition. This is either that related to the default chosen operating system,
+  or first one found when the chosen default operating system is not from Apple or does not have
+  a recovery partition. Hold the \texttt{CMD+R} key combination down to choose this option.
   \end{itemize}
 
-  \emph{Note 1}: Activated \texttt{KeySupport}, \texttt{OpenUsbKbDxe}, or similar driver is required
-  for key handling to work. On several types of firmware, it is not possible to get all the key functions.
+  \emph{Note 1}: The \texttt{KeySupport}, \texttt{OpenUsbKbDxe}, or similar drivers are required for key
+  handling. However, not all of the key handling functions are available on several types of firmware.
```

Comment:
These functions may not be implemented in the firmware. E.g. `KeySupport` builds this on top firmware drivers. The original description is more accurate. `can be implemented on several types of firmware` perhaps.